.github/BOTMETA.yml

```
@@ -717,7 +717,7 @@ files:
       - clustering
       - k8s
   $module_utils/identity/keycloak/:
-     maintainers: eikef
+     maintainers: eikef, ndclt
```

Needs to be a space-separated list
```suggestion
     maintainers: eikef ndclt
```

---

Thanks. Corrected in db4d7e919e6731ead8ee2a085e49f03eb38d840a.